VKApp/VKApp/Services/VKNetworkService.swift

```diff
@@ -125,38 +148,17 @@ final class VKService {
         return urlComponents
     }
 
-    // MARK: - Private Methods
-
-    private func saveGroupVKData(_ groupVK: [ItemGroupVK]) {
-        do {
-            let realm = try Realm()
-            realm.beginWrite()
-            realm.add(groupVK)
-            try realm.commitWrite()
-        } catch {
-            print(error)
-        }
-    }
-
-    private func saveFriendsData(_ friends: [ItemPerson]) {
-        do {
-            let realm = try Realm()
-            realm.beginWrite()
-            realm.add(friends)
-            try realm.commitWrite()
-        } catch {
-            print(error)
-        }
-    }
-
-    private func savePhotosData(_ photos: [ItemPhoto]) {
-        do {
-            let realm = try Realm()
-            realm.beginWrite()
-            realm.add(photos)
-            try realm.commitWrite()
-        } catch {
-            print(error)
+    func setupImage(urlPath: String?, imageView: UIImageView) {
+        guard
+            let urlPath = urlPath,
+            let url = URL(string: urlPath)
+        else { return }
+        DispatchQueue.global().async {
+            let data = try? Data(contentsOf: url)
+            DispatchQueue.main.async {
+                guard let data = data else { return }
+                imageView.image = UIImage(data: data)
+            }
         }
```

Comment:
```suggestion
    func fetchData(urlPath: String?) -> Data? {
        guard
            let urlPath = urlPath,
            let url = URL(string: urlPath)
        else { return }
        DispatchQueue.global().async {
            let data = try? Data(contentsOf: url)
               return data
            }
        }
```
часть c UI вынести в extension UIImageView и там дергать метод получения данных